index.html

```diff
@@ -8,7 +8,7 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;700&display=swap" rel="stylesheet">
     <script src="https://kit.fontawesome.com/1bc3a7e061.js" crossorigin="anonymous"></script>
-    <title>Mobile first Portfolio</title>
+    <title>My Portfolio | Belay Birhanu</title>
```

Comment:
Good job @belaymit 👍,  according to the accessibility standards, I think it will be better to use 'Belay Birhanu's Portfolio' instead of starting it with 'My portfolio...'